docs/tutorial.rst

```diff
@@ -242,5 +242,151 @@ effective population size from the **TODO add crossref to catalog section**
 .. :ref:`sec_catalog_homo_sapiens_genome`
 
 
+************************************************
+Analysis: Genetic divergence between populations
+************************************************
 
+In this tutorial, we will simulate some samples of human chromosomes
+from different populations,
+and then estimate the genetic divergence between each population pair. 
 
+First, let's use the ``--help-models`` option to see the selection of demographic
+models available to us:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-models
+
+This prints detailed information about all of the available models to
+the terminal.
+In this tutorial, we will use the model of African-American admixture from
+`2011 Browning et al <http://dx.doi.org/10.1371/journal.pgen.1007385>`_.  
+From the help output (or the :ref:`Catalog <sec_catalog>`),
+we can see that this model has id ``america``,
+and allows samples to be drawn from 4 contemporary populations representing African,
+European, Asian and African-American groups.
+
+Using the ``--help-genetic-maps`` option, we can also see what recombination maps
+are available:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-genetic-maps
+
+Let's go with ``HapmapII_GRCh37``.
+The next command simulates 4 samples of chromosome 1 from each of the four
+populations, and saves the output to a file called ``afr-america-chr1.trees``.
+For the purposes of this tutorial, we'll also specify a random seed using the
+``-s`` option.
+(Note: This took around 8 minutes to run on a laptop.)
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr1 -o afr-america-chr1.trees -s 13 -g HapmapII_GRCh37\
+    --model america 4 4 4 4
+
+We should now have a file called ``afr-america-chr1.trees``.
+Our work with ``stdpopsim`` is done; we'll now switch to a Python console and import
+the ``tskit`` package to load and analyse this simulated tree sequence file.
+
+.. code-block:: python
+
+    >>> import tskit
+    >>> ts = tskit.load(afr-america-chr1.trees)
+
+Recall that `genetic divergence` is the probability that two randomly sampled
+chromosomes differ at a nucleotide base.
+For a given pair of populations, a pair-specific divergence value is obtained
+by randomly sampling one chromosome from each population.
+These quantities can be estimated directly from our sample using tskit's
+inbuilt `TreeSequence.diversity <https://tskit.readthedocs.io/en/latest/python-api.html#tskit.TreeSequence.divergence>`_ method.
+
+The documentation for this method specifies the following required input:
+
+    sample_sets (list) – A list of lists of Node IDs, specifying the groups of individuals to compute the statistic with.
```

Comment:
This formatting is a little confusing to me.  Also, are `indexes` required?

I think I would just say:
"This method requires `sample_sets` and `indexes`. Let's think..."
And let them look at the linked docs themselves for more detail.

Reply:
`indexes` are required if you care calculating divergences on more than just a single pair of sets. I think the tskit docs are a bit confusing on that one